Sources/GRPCInterceptors/ClientTracingInterceptor.swift

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import GRPCCore
+import Tracing
+
+/// A client interceptor that injects tracing information into the request.
+///
+/// The tracing information will be taken from the current `ServiceContext`, and injected into the request's
+/// metadata, to be picked up by the server-side ``ServerTracingInterceptor``.
```

Comment:
Total nit, but metadata is a gRPC concept but the `ServerTracingInterceptor` is specific to grpc-swift. I think the latter part of the sentence should be a separate sentence.